__tests__/app.test.js

```diff
@@ -156,5 +156,61 @@ describe("/api/articles", () => {
         const { message } = body;
         expect(message).toBe("Endpoint not found");
       });
+  });
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET 200: Should return an array of comments for the given article ID with the required properties", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        comments.forEach((comment) => {
```

Comment:
When using a .forEach, you must check that the length is at least greater than 0. If the length of the array is 0, it will skip the .forEach, meaning that it will auto-pass :)